src/PaymentGateways/Actions/RegisterPaymentGateways.php

```diff
@@ -17,7 +18,7 @@ class RegisterPaymentGateways
      */
     public $gateways = [
         // When complete, the Test Gateway will eventually replace The legacy Manual Gateway.
-        //TestGateway::class,
+        TestGateway::class,
```

Comment:
Do we need to comment this again before merging?

Reply:
Yup good catch 9ebc74f4